src/tests/Routes/Routes.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import { MemoryRouter } from 'react-router';
+import { Routes } from '../../Routes/Routes.component';
+import { VideosContext } from '../../providers/Videos/Videos.provider';
+
+describe('Routes', () => {
+    test('should render correctly', () => {
+        const wrapper = shallow(
+            <MemoryRouter>
+                <Routes></Routes>
```

Comment:
No hay nada malo aquí, pero si se cierra el componente allí mismo mejor cambiarlo.
```suggestion
                <Routes/>
```